### DISCUSSION TITLE
Lesson 6 - eth_usd_price_feed

### DISCUSSION THREAD

**Discussion by realglass** [smartcontractkit/full-blockchain-solidity-course-py#1565] [Q&A] [answered]
Hi guys! Probably this is some kind of syntax error I can't find or figure out what I am doing wrong. Any help?

![image](https://user-images.githubusercontent.com/101928609/177028469-f5f84a41-cf5f-4c70-a6f9-5be00b39a279.png)
![image](https://user-images.githubusercontent.com/101928609/177028486-bf4cafac-4002-4ffc-b3d2-d4211d303f8a.png)
![image](https://user-images.githubusercontent.com/101928609/177028510-02e3a78d-130e-40d9-be2a-6cab509c34fa.png)

**Comment by cromewar** [comment, marked as answer]
Hello @realglass 
Well all it depends on which network are you using to deploy? as you an see you just have the `eht_usd_price_feed` for rinkeby and mainnet-fork-dev, so if you deploy on development or ganache-local it'll search on the brownie-config and is not finding it.

**Reply by realglass** [reply]
hi! thank you for your reply! managed to fix it

**Reply by Jawadirshad** [reply]
> hi! thank you for your reply! managed to fix it
Hi can you please Guide me how you fixed it, i am facing the same issue
